maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -874,9 +873,9 @@ private static <T> List<T> reverse(List<T> list) {
     }
 
     private List<File> parseExtClasspath(CliRequest cliRequest) {
-        String extClassPath = cliRequest.userProperties.getProperty(EXT_CLASS_PATH);
+        String extClassPath = cliRequest.userProperties.getProperty(Constants.MAVEN_EXT_CLASS_PATH);
         if (extClassPath == null) {
-            extClassPath = cliRequest.systemProperties.getProperty(EXT_CLASS_PATH);
+            extClassPath = cliRequest.systemProperties.getProperty(Constants.MAVEN_EXT_CLASS_PATH);
```

Comment:
Can we add a note whether this should still be system property? If we are already in Java then we obviously don't need a system property at all, only user one.

See: https://github.com/apache/maven/pull/1595/files#diff-b8b814f5b6b3855ae79dc45a0266b94871193dcef42803c316e07409e94af35cR1404

Reply:
I think we can keep a user properties only.
What about adding a warning if the user uses this system property ? We could direct him to use a user property instead ?

Reply:
@michael-o I've added a warning if a system property is used.